pkg/receive/handler_test.go

```diff
@@ -1293,7 +1293,7 @@ func Heap(dir string) (err error) {
 		return err
 	}
 
-	f, err := os.Create(filepath.Join(dir, "impr5-go1.16.3.pprof"))
+	f, err := os.Create(filepath.Join(dir, "errimpr1-go1.16.3.pprof"))
```

Comment:
Is it only for testing?

Reply:
Yes, I provably should remove those before merging but those are super useful for checking leaks (: